### PR TITLE
Adding Bigtable row filters for boolean flags.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: gcloud/bigtable/row.py [hunk 1]
```diff
@@ -253,3 +324,21 @@ def to_pb(self):
         :returns: The converted current object.
         """
         return data_pb2.RowFilter(cells_per_column_limit_filter=self.num_cells)
+
+
+class StripValueTransformerFilter(_BoolFilter):
```
Comment (tseaver) [inline gcloud/bigtable/row.py]: Why not keep it with the other `_BoolFilter` derivatives?  Are you synching with an order in the protobuf spec, or the docs?

**Comment by tseaver** [comment]
LGTM except for my ordering question.

File: gcloud/bigtable/row.py (hunk 1 above)
Comment (dhermes) [inline gcloud/bigtable/row.py]: I'm not sure which docs you're referring to, but the goal is to keep the order the same as in [the protobuf spec][1]

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/03723524694e21ebc14b21b6371d29e316d04130/bigtable-protos/src/main/proto/google/bigtable/v1/bigtable_data.proto#L321-L404

Comment (tseaver) [inline gcloud/bigtable/row.py]: OK, then the order in `test_row` should match as well.

Comment (dhermes) [inline gcloud/bigtable/row.py]: On it. I debated doing that but figured putting them at the bottom might make them easier to review. Will ping once done.

**Comment by dhermes** [comment]
@tseaver Will merge on Travis success

**Comment by dhermes** [comment]
For some reason Travis didn't trigger.

**Pull request merged**